README.md

```diff
@@ -33,7 +34,7 @@ Don't forget the `enctype="multipart/form-data"` in your form.
 
 ```html
 <form action="/profile" method="post" enctype="multipart/form-data">
-  <input type="file" name="avatar" />
+    <input type="file" name="avatar" />
```

Comment:
I think we should keep the indentation at 2 spaces to match the JS code, and all the other readme languages

```suggestion
  <input type="file" name="avatar" />
```